requirements-dev.txt

```diff
@@ -30,3 +30,4 @@ black
 flake8
 mypy
 pytensor>=2.28.0
+mkl-fft; sys_platform != "darwin"
```

Comment:
What does this mean? 

If I look at the documentation of `mkl_fft` I see that they suggest that to install the library with `pip` one has to do 

```
   python -m pip install --index-url https://software.repos.intel.com/python/pypi --extra-index-url https://pypi.org/simple mkl_fft
```

but you seem to install it from the standard url... is it because of this: https://pypi.org/project/mkl-fft/?

And why not on mac? I guess the macos tests run fin as `mkl_fft` is not installed and so it falls back to numpy?